components/styled/table/src/organisms/questionnaire-table-methods.ts

```diff
@@ -114,21 +123,29 @@ const mapQuestionnaireResponsesIntoDataEntities = (
 
 const recursivelyMapResponseItemsOntoData = (
   items: QuestionnaireResponseItem[],
-  dataEntity: DataEntity
+  dataEntity: DataEntity,
+  status: QuestionnaireResponseStatus
 ): DataEntity => {
   let updatedDataEntity = { ...dataEntity }
   items.forEach((item) => {
     const firstAnswer = item.answer ? item.answer[0] : undefined
 
     if (item.linkId && firstAnswer) {
+      const props = createCellPropsForAnswer(
+        firstAnswer,
+        false,
+        status === QuestionnaireResponseStatus.EnteredInError ? true : undefined
+      )
       updatedDataEntity[item.linkId] = {
-        text: createCellPropsForAnswer(firstAnswer, false).text,
+        text: props.text,
```

Comment:
Is removing the call to `createCellPropsForAnswer` OK here? Just wanting to make sure you can shortcut the logic it was previously doing to generate `text`